cid-redirects.json

```
@@ -3091,13 +3091,13 @@
   "/Manage/Partitions/Manage_Indexes_with_Variable_Retention": "/docs/manage/partitions/manage-indexes-variable-retention",
   "/Manage/Collection/Processing_Rules": "/docs/send-data/collection/processing-rules",
   "/Manage/Collection/Processing_Rules/Create_a_Processing_Rule": "/docs/send-data/collection/processing-rules/create-processing-rule",
-  "/Manage/Scheduled-Views": "/docs/manage/scheduled-views",
-  "/Manage/Scheduled-Views/Add-a-Scheduled-View": "/docs/manage/scheduled-views/add-scheduled-view",
-  "/Manage/Scheduled-Views/Pause-or-Disable-Scheduled-Views": "/docs/manage/scheduled-views/pause-disable-scheduled-views",
-  "/Manage/Scheduled-Views/Scheduled-View-Lag-Time": "/docs/manage/scheduled-views/scheduled-view-lag-time",
-  "/Manage/Scheduled-Views/Run-a-Search-Against-a-Scheduled-View": "/docs/manage/scheduled-views/run-search-against-scheduled-view",
-  "/Manage/Scheduled-Views/Scheduled-Views-Best-Practices-and-Examples": "/docs/manage/scheduled-views/scheduled-views-best-practices",
-  "/Manage/Scheduled-Views/View_a_List_of_Scheduled_Views": "/docs/manage/scheduled-views/view-list-scheduled-views",
+  "/Manage/Scheduled-Views": "/docs/manage/views",
+  "/Manage/Scheduled-Views/Add-a-Scheduled-View": "/docs/manage/views/add-scheduled-view",
+  "/Manage/Scheduled-Views/Pause-or-Disable-Scheduled-Views": "/docs/manage/views/pause-disable-scheduled-views",
+  "/Manage/Scheduled-Views/Scheduled-View-Lag-Time": "/docs/manage/views/scheduled-view-lag-time",
+  "/Manage/Scheduled-Views/Run-a-Search-Against-a-Scheduled-View": "/docs/manage/views/run-search-against-scheduled-view",
+  "/Manage/Scheduled-Views/Scheduled-Views-Best-Practices-and-Examples": "/docs/manage/views/scheduled-views-best-practices",
```

Does "scheduled" need to be removed from this link?

```suggestion
  "/Manage/Scheduled-Views/Scheduled-Views-Best-Practices-and-Examples": "/docs/manage/views/best-practices",
```